ops/services/app_service/metabase-readonly/_data.tf

```diff
@@ -0,0 +1 @@
+data "azurerm_client_config" "current" {}
```

Comment:
I don't think is necessary, since it's never used in `ops/services/app_service/metabase-readonly/metabase.tf`.

Reply:
Removed this file.